internal/pkg/template/templates/workloads/partials/cf/vpc-connector.yml

```diff
@@ -11,6 +11,16 @@ ServiceSecurityGroup:
       - Key: Name
         Value: !Sub 'copilot-${AppName}-${EnvName}-${WorkloadName}'
 
+EnvironmentSecurityGroupIngressFromServiceSecurityGroup:
+  Type: AWS::EC2::SecurityGroupIngress
+  Properties:
+    Description: Ingress from the app runner service security group.
```

Comment:
Can we add this as the Copilot metadata so that it shows up in the progress tracker?
```suggestion
    Description: Allow ingress from the app runner service to services in your environment
```